src/main/kotlin/no/nav/tilleggsstonader/soknad/arkivering/ArkiveringService.kt

```diff
@@ -11,25 +11,25 @@ import java.util.UUID
 @Service
 class ArkiveringService(
     private val integrasjonerClient: IntegrasjonerClient,
-    private val søknadService: SøknadService,
+    private val skjemaService: SkjemaService,
     private val vedleggRepository: VedleggRepository,
 ) {
     fun journalførSøknad(
         søknadId: UUID,
         callId: String,
     ) {
```

Comment:
Denne kan kanskje brukes på kjørelister også? I så fall
```suggestion
    fun journalførSkjema(
        skjemaId: UUID,
        callId: String,
    ) {
```